src/WormholeEndpoint.sol

```diff
@@ -79,6 +84,17 @@ abstract contract WormholeEndpoint is Endpoint, IWormholeEndpoint, IWormholeRece
         }
     }
 
+    function _getSpecialRelayingEnabledChainsStorage()
+        internal
+        pure
+        returns (mapping(uint16 => bool) storage $)
```

Comment:
id suggest using the native EVM word size here (uint256) instead of a bool. saves an extra SLOAD when reading from storage later

Reply:
Can do. Should we also update all the other storage getter to return uint256 instead of bool, such as `_getWormholeRelayingEnabledChainsStorage`?

Reply:
Yea I think so